cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -144,28 +149,59 @@ def test_button_crud(appliance, dialog, request, buttongroup, obj_type):
         1143019
         1205235
     """
-    button_gp = buttongroup(obj_type)
-    button = button_gp.buttons.create(
-        text=fauxfactory.gen_alphanumeric(),
-        hover=fauxfactory.gen_alphanumeric(),
+
+    # Create button under button group
+    button = button_group.buttons.create(
+        text=fauxfactory.gen_alphanumeric(start="btn_"),
+        hover=fauxfactory.gen_alphanumeric(start="hvr_"),
+        icon_color="#ff0000",
```

Comment:
Thoughts on making this color the default?

Reply:
good point